doc/source/installation/digital_alliance.rst

```diff
@@ -69,16 +72,20 @@ We can compile ``dealii`` in the ``$HOME/dealii/build`` folder, by defining the
 
 .. tip::
```

Comment:
I feel like this is more like "important" than a tip, but that's all fine

Reply:
Yeah that's true, not sure what would be the alternative word though...